sklearn/utils/arrayfuncs.pyx

```diff
@@ -4,29 +4,24 @@ Small collection of auxiliary functions that operate on arrays
 """
 
 cimport numpy as cnp
-import  numpy as np
+import numpy as np
 from cython cimport floating
 from libc.math cimport fabs
 from libc.float cimport DBL_MAX, FLT_MAX
 
 from ._cython_blas cimport _copy, _rotg, _rot
 
-ctypedef cnp.float64_t DOUBLE
 
-
-cnp.import_array()
```

Comment:
If the previous suggestions does not resolve the problem, can you revert this line?